[2022-10-12 04:56:07.394] [remoteagent] [info] Extension host agent started.
[2022-10-12 04:56:08.786] [remoteagent] [info] [127.0.0.1][0c92084d][ManagementConnection] New connection established.
[2022-10-12 04:56:08.789] [remoteagent] [info] [127.0.0.1][e97e4f1f][ExtensionHostConnection] New connection established.
[2022-10-12 04:56:08.804] [remoteagent] [info] [127.0.0.1][e97e4f1f][ExtensionHostConnection] <503179> Launched Extension Host Process.
[2022-10-12 04:56:20.560] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-12 04:56:20.562] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateIcon (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:3046)","    at Object.call (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-12 04:59:49.116] [remoteagent] [info] [127.0.0.1][90aec714][ManagementConnection] New connection established.
[2022-10-12 04:59:49.532] [remoteagent] [info] [127.0.0.1][22f95366][ExtensionHostConnection] New connection established.
[2022-10-12 04:59:49.574] [remoteagent] [info] [127.0.0.1][22f95366][ExtensionHostConnection] <503477> Launched Extension Host Process.
[2022-10-12 05:41:32.256] [remoteagent] [info] [127.0.0.1][22f95366][ExtensionHostConnection] The client has reconnected.
[2022-10-12 05:41:32.638] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateIcon (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:3046)","    at Object.call (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-12 05:41:32.640] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-12 05:41:32.835] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-12 05:42:25.525] [remoteagent] [info] [127.0.0.1][90aec714][ManagementConnection] The client has reconnected.
[2022-10-12 05:42:27.118] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/aleksie/.vscode-server/bin/129500ee4c8ab7263461ffe327268ba56b9f210d/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-12 05:46:33.776] [remoteagent] [info] [127.0.0.1][e97e4f1f][ExtensionHostConnection] <503179> Extension Host Process exited with code: 0, signal: null.
